src/frontend/src/catalog/table_catalog.rs

```diff
@@ -138,7 +138,7 @@ impl TableCatalog {
         self.distribution_key.as_ref()
     }
 
-    pub fn to_state_table_prost(&self) -> ProstTable {
+    pub fn to_internal_table_prost(&self) -> ProstTable {
```

Comment:
why we need change this name?

Reply:
currently, only internal tables need to fill in a placeholder, while mv does not need this.